### PR TITLE
chore(editorconfig): Added new rule for scss in editorconfig

### PR DESCRIPTION
**Pull request by narukaioh**
the proposal here is that the scss files should be indented by spaces, motivating the need for many levels of indentation.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -7,3 +7,7 @@ indent_style = tab
 indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
+
+[*.scss]
+indent_style = space
```
Comment (aMediocreDad) [inline .editorconfig:12]: Keep tabs, but let indent_size be 2.

This should also be reflected in Prettierrc

Comment (narukaioh) [reply .editorconfig]: I removed the line 12 on this PR and added tabWidth in prettierrc

Comment (aMediocreDad) [reply .editorconfig]: Thanks!

**Comment by aMediocreDad** [comment]
👍 Looks good! Merging and closing:)

I merged with dev branch since it sees more rapid development.

**Pull request merged**